Assets/Scripts/Export/ImportMaterialCollector.cs

```diff
@@ -92,6 +92,25 @@ public void AddSvgIem(Material unityMaterial)
   }
 #endif
 
+        // Used for GLTFast
+        public void Add(Material unityMaterial, bool transparent)
```

Comment:
Did you mean to do something with `transparent`?

Reply:
Nope. That was a leftover from a last minute change. It's now inferred directly from the material's renderQueue value